libnmstate/netapplier.py

```diff
@@ -116,9 +116,7 @@ def _apply_ifaces_state(desired_state, verify_change, commit,
                 _add_interfaces(desired_state.interfaces,
                                 current_state.interfaces)
             with _setup_providers():
-                current_state = state.State(
-                    {Constants.INTERFACES: netinfo.interfaces()}
-                )
+                current_state = state.State(netinfo.show())
```

Comment:
Also seems to be an unrelated change

Reply:
This is related as State now holds the route configuration also.